src/common.js

```diff
@@ -123,9 +123,11 @@ function resolvePackages(pkgs) {
   for (var p in pkgs) {
     var value = pkgs[p];
     // TODO package fallback support
-    if (typeof value !== 'string')
-      continue;
-    outPkgs[resolveIfNotPlainOrUrl(p) || p] = value;
+    if (typeof value !== 'string') {
+      outPkgs[resolveIfNotPlainOrUrl(p) || p] = value.filter(v => !v.startsWith('std:'))[0];
```

Comment:
`value.find(v => !v.startsWith('std:'))` seems shorter here?

Also in the case where the value is undefined we shouldn't be assigning. So perhaps rather something like:

```js
if (Array.isArray(value))
  value = value.find(v => !v.startsWith('std:'));
if (typeof value === 'string')
  outPkgs[resolveIfNotPlainOrUrl(p) || p] = value;
```

as the whole thing.